pyproject.toml

```diff
@@ -0,0 +1,195 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2024 Collabora Limited
+# Author: Jeny Sadadia <jeny.sadadia@collabora.com>
+
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "kernelci"
+version = "1.0"
+description = "KernelCI core functions"
+readme = "README.md"
+requires-python = ">=3.8"
+license = {text = "LGPL-2.1-or-later"}
+dependencies = [
+  "azure-storage-file-share==12.13.0",
+  "bson==0.5.10",
+  "click==8.1.3",
+  "cloudevents==1.9.0",
+  "docker==6.1.2",
+  "jinja2==3.1.3",
+  "kubernetes==26.1.0",
+  "paramiko==3.4.0",
+  "pydantic==1.10.5",
+  "pyelftools==0.29",
+  "pytest==7.4.0",
+  "pyyaml==6.0",
+  "requests==2.31.0",
+  "scp==0.14.5",
+  "toml==0.10.2",
+]
+authors = [
+  {name = "Guillaume Tucker" , email = "guillaume.tucker@collabora.com"},
+  {name = "Tyler Baker" , email = "tyler.baker@linaro.org"},
+  {name = "Denys Fedoryshchenko" , email = "denys.f@collabora.com"},
+  {name = "Mark Brown" , email = "broonie@kernel.org"},
+  {name = "Kevin Hilman" , email = "khilman@baylibre.com"},
+  {name = "Matt Hart" , email = "matthew.hart@linaro.org"},
+  {name = "Jeny Sadadia" , email = "jeny.sadadia@collabora.com"},
+  {name = "Quentin Schulz" , email = "quentin.schulz@free-electrons.com"},
+  {name = "Matt Hart" , email = "github@blacklabsystems.com"},
+  {name = "Michal Galka" , email = "michal.galka@collabora.com"},
+  {name = "Corentin LABBE" , email = "clabbe@baylibre.com"},
+  {name = "Anders Roxell" , email = "anders.roxell@linaro.org"},
+  {name = "Ana Guerrero Lopez" , email = "ana.guerrero@collabora.com"},
+  {name = "Nícolas F. R. A. Prado" , email = "nfraprado@collabora.com"},
+  {name = "kernelci.org bot" , email = "bot@kernelci.org"},
+  {name = "Tyler Baker" , email = "forcedinductionz@gmail.com"},
+  {name = "Laura Nao" , email = "laura.nao@collabora.com"},
+  {name = "Miguel Ojeda" , email = "ojeda@kernel.org"},
+  {name = "khilman" , email = "khilman@users.noreply.github.com"},
+  {name = "Michael Grzeschik" , email = "m.grzeschik@pengutronix.de"},
+  {name = "Alexandra Pereira" , email = "alexandra.pereira@collabora.com"},
+  {name = "Alice Ferrazzi" , email = "alice.ferrazzi@miraclelinux.com"},
+  {name = "Adrian Ratiu" , email = "adrian.ratiu@collabora.com"},
+  {name = "Chris Paterson" , email = "chris.paterson2@renesas.com"},
+  {name = "Guillaume Tucker" , email = "gtucker@gtucker.io"},
```

Comment:
Ah I see my current email address is here so the other @collabora.com entry can just be removed.